pkg/provider/azure.go

```diff
@@ -402,60 +405,40 @@ func (az *Cloud) InitializeCloudFromConfig(ctx context.Context, config *config.C
 		az.ARMClientConfig.UserAgent = fmt.Sprintf("kubernetes-cloudprovider/%s", k8sVersion)
 	}
 
```

Comment:
It will log in L400 @feiskyer 

https://github.com/kubernetes-sigs/cloud-provider-azure/pull/8730#discussion_r2011439029